src/RuntimeStoreVerifier/RuntimeStoreVerifier.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>RuntimeStoreVerifier</id>
```

Comment:
FYI - one of the near-team goals of the "Build KoreBuild" work I'm doing is that we will not longer install these kinds of tools via NuGet. Instead, they will be bundled as tasks in KoreBuild. But this is fine for now as the 2.0.0 Coherence-Signed branch still uses pre-flight restore.